main.cpp

```diff
@@ -0,0 +1,136 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+class Inventory {
+private:
+    string products[100];
+    int itemCount;
+
+public:
+    Inventory() : itemCount(0) {}
```

Comment:
In your code, you have a mixture of responsibilities related to inventory management and displaying cuisine information.